_posts/2020-06-25-changelog-125.markdown

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #125"
+date: 2020-06-25 19:26:11 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5535 -->
+* Support debugging with MSVC toolchain in CLion 2020.2
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5586 -->
+* Add `Introduce local variable` intention (by [@Kobzol])
+
+    TODO: GIF
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5626 -->
+* Implement `Move File` refactoring
+
+    TODO: GIF?
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5614 -->
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5601 -->
+* Add intentions to convert a method call to UFCS and vice versa (by [@Kobzol])
+
+    TODO: GIF
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5550 -->
+* Add the quick fix to convert an immutable reference to a mutable one (by [@Kobzol])
+
+    TODO: GIF
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5557 -->
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5561 -->
+* Add quick fixes to remove unused variable / parameter (by [@Kobzol])
+
+    TODO: GIF
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5609 -->
+* Update icons for Rust items
+
+    TODO: PNG (old / new)?
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/4640 -->
+* Enable intelligent highlighting inside macro call bodies
```

Comment:
Let's note that it works only with the new expansion engine and add instructions about how to enable it.